applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                        <item value="front_top_right"/>
+                        <item value="front_bottom_right"/>
+                        <item value="front_bottom_left"/>
+                        <item value="back_top_left"/>
+                        <item value="back_top_right"/>
+                        <item value="back_bottom_right"/>
+                        <item value="back_bottom_left"/>
+                    </enumeration>
+                </field>
+                <field name="x_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing x-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="y_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing y-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="z_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing z-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+            </group>
+        </group>
+        <group name="measurement" type="NXobject" optional="true">
+            <!--the choice if a concept filling content in NXfabrication is recommended or optional
+was made such that all for all those components which are typically add-ons in a
+microscope it is more likely that individuals will have bought different third-party tools
+and therefore, for these typically more optional add-ons NXfabrication is recommended
+for others commercial microscopes often come with model-specific predefined parts
+hence using optional is sufficient-->
```

Comment:
Developer comments needed?

Reply:
Have removed them, if needed they are part of the history